step.yml

```diff
@@ -336,6 +336,21 @@ inputs:
     value_options:
     - "false"
     - "true"
+- use_legacy_xcresult_extraction_method: "false"
+  opts:
+    category: Debugging
+    title: Legacy extraction method for Xcode test results
+    summary: |-
+      The Step will try to extract the test results with the older extraction method.
```

Comment:
Would it make sense to be more verbose about what legacy/older extraction method means? Like mentioning the related Xcode version. I can imagine new xcresult formats and new extraction methods in the feature.

Reply:
Great idea. I have updated it.